rig/machine_control/sdram_file.py

```diff
@@ -0,0 +1,84 @@
+class SDRAMFile(object):
+    def __init__(self, machine_controller, x, y,
+                 start_address=0x60000000,
+                 end_address=0x6fffffff):
```

Comment:
Magic default numbers! In my (somewhat strongly held) opinion, the defaults should be the 4mb part near the start of SDRAM which is defined as a "free-for-all" (`sv->sdram_base` to `sv->sdram_heap`.). This encourages "modern" behaviour, i.e. using the memory allocator when using blocks of SDRAM.

Bonus points for alternative constructors which look like "malloc" or which take a tag number as argument.